addon/templates/components/frost-sidebar.hbs

```diff
@@ -1,20 +1,20 @@
-{{#liquid-if isOpen class="container" }}
-  <div class="open-container">
-    <div class="pod">
+{{#liquid-if isOpen}}
+  <div class="frost-sidebar-open" data-test={{hook (concat hook '-sidebar')}}>
+    <div class="frost-sidebar-content-container">
       {{#frost-scroll}}
-        <div class="pod-content">
-          {{yield}}
+        <div class="frost-sidebar-content" data-test={{hook (concat hook '-sidebar-content')}}>
+          {{component content}}
         </div>
       {{/frost-scroll}}
     </div>
-    <div class="tab tab-open" {{action (action 'toggleSidebar')}}>
-      {{frost-icon class="icon" icon="frost/sidebar-layer-open"}}
+    <div class="frost-sidebar-tab frost-sidebar-tab-open" {{action onToggle}} data-test={{hook (concat hook '-sidebar-tab')}}>
```

Comment:
Yes, always feel like it to do this kind of improvements :) Clarity is my best friend in code :P